### PR TITLE
feat: Add interpreter option to `wait_for_cluster_cmd`

### PR DESCRIPTION
**Pull request by avoidik**
# PR o'clock

## Description

Allow to configure `interpreter` item for the `wait_for_cluster_cmd` command. Potential improvement into compatibility with Windows

https://github.com/terraform-aws-modules/terraform-aws-eks/issues/680

### Checklist

- [x] Change added to CHANGELOG.md. All changes must be added and breaking changes and highlighted
- [ ] CI tests are passing
- [x] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

**Comment by avoidik** [comment]
rel.
https://github.com/terraform-aws-modules/terraform-aws-eks/pull/639
https://github.com/terraform-aws-modules/terraform-aws-eks/pull/701

**Comment by avoidik** [comment]
```hcl
module "eks" {
  source  = "terraform-aws-modules/eks/aws"
  # windows workaround
  wait_for_cluster_interpreter = ["c:/git/bin/sh.exe", "-c"]
  wait_for_cluster_cmd         = "until curl -sk $ENDPOINT >/dev/null; do sleep 4; done"
}
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [[v10.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v10.0.0...HEAD)] - 2020-xx-xx
 
-- Write your awesome change here (GH-xxxx)
+- Allow to configure interpreter for the `wait_for_cluster_cmd` command (by @avoidik)
```
Comment (barryib) [inline CHANGELOG.md]: Please remove your changes from changelog. It's now auto generated during release.

Comment (avoidik) [reply CHANGELOG.md]: done

**Comment by amitsehgal** [comment]
can this variable be added at node group, worker group level and not at cluster level ?

**Comment by Type1J** [comment]
Doesn't this break Linux nodes?

**Comment by avoidik** [comment]
@Type1J it is optional parameter, if not specified safe defaults will be used instead

**Comment by Type1J** [comment]
I meant in the same cluster. I haven't tried it, but as @amitsehgal implied, the worker group could be Linux or Windows, and since this is specified at the module level, it seems that this parameter would affect both worker groups, so only 1 worker group would have the correct interpreter. Again, I haven't tried it, yet. I'm evaluating this module for use in a mixed (Linux and Windows) EKS cluster, so I'd like to understand it.

**Comment by avoidik** [comment]
@Type1J not sure how it supposed to affect individual worker groups, the command being executed locally to confirm k8s api readiness

**Comment by Type1J** [comment]
Oh. Thanks for the clarification.

**Pull request merged**